controller/test-files/event/list/ok-description.res.payload.golden.json

```diff
@@ -3,7 +3,10 @@
     {
       "attributes": {
         "name": "system.description",
-        "newValue": null,
+        "newValue": {
+          "content": "# Description is modified1",
```

Comment:
I am not sure if we should send the description to the client when they won't be using it. A description is the most commonly updated field. We will end up sending a huge amount of unnecessary data to the client.

I agree one field shouldn't be dealt with differently but in this case, the special handling provides improved performance. I feel this is a valuable trade-off

Reply:
I don't feel this way. This is pre-mature optimization and potentially a field like `steps-to-reproduce` (which is also a markup field) is more often updated than the description.